test/rotoscope_test.rb

```diff
@@ -288,6 +288,19 @@ def test_trace_ignores_calls_if_blacklisted
     assert_frames_consistent contents
   end
 
+  def test_trace_saves_called_context_if_called_from_blacklisted_path
+    dependency = FixtureInner.new
+    contents = rotoscope_trace(flatten: true, blacklist: ['/rotoscope_test.rb']) do
```

Comment:
What does the unflattened output look like?

Reply:
```
[
  {:event=>"call", :entity=>"FixtureInner", :method_name=>"sum", :method_level=>"instance", :filepath=>"/fixture_inner.rb", :lineno=>-1},
  {:event=>"return", :entity=>"FixtureInner", :method_name=>"sum", :method_level=>"instance", :filepath=>"/fixture_inner.rb", :lineno=>-1},
]
```

which isn't ideal, since that doesn't allow us to get more context from using the unflattened output.